ceph.spec.in

```diff
@@ -1199,6 +1199,12 @@ make DESTDIR=%{buildroot} install
 # we have dropped sysvinit bits
 rm -f %{buildroot}/%{_sysconfdir}/init.d/ceph
 popd
+
+%if 0%{with seastar}
+# package crimson-osd with the name of ceph-osd
+install -m 0755 %{buildroot}%{_bindir}/crimson-osd %{buildroot}%{_bindir}/ceph-osd
```

Comment:
IIUC even in the migration scenario crimson will live in a dedicated container, so there shouldn't be any conflict on binary names with the classic OSD – neither if classical was pulled as a package nor another container. Hmm, that looks nice.